src/app/dashboard/loading.tsx

```
@@ -0,0 +1,14 @@
+import { Spinner } from '@/components/global/loader/spinner'
+import React from 'react'
+
+type Props = {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)